### ISSUE TITLE
Refreshing devices after automation

### ISSUE THREAD

**Issue by JimBobSD** [sibartlett/homebridge-wink3#13]
I noticed another odd issue.  After running an automation, it performs correctly.  In this case, it was an automation that turned off certain lights.  However, the Home App still shows (for example) that the lights are "on."  The Home App showed this for about 30  minutes until the Wink3 log showed "refreshing devices/Devices refreshed."  At that point, the lights correctly showed as "off."  Here's the log:

```
[5/22/2017, 9:27:13 AM] [Wink] Sending update: Kitchen Cabinets (binary_switch/455994) { powered: false }
[5/22/2017, 9:27:13 AM] [Wink] Sending update: Kitchen Main (light_bulb/2645199) { powered: false }
[5/22/2017, 9:27:13 AM] [Wink] Sending update: Kitchen Nook Ceiling (light_bulb/2645178) { powered: false }
[5/22/2017, 9:27:13 AM] [Wink] Sending update: Kitchen Nook Fixture (light_bulb/2645176) { powered: false }
[5/22/2017, 9:27:14 AM] [Wink] Update sent successfully: Kitchen Nook Fixture (light_bulb/2645176)
[5/22/2017, 9:27:14 AM] [Wink] Update sent successfully: Kitchen Cabinets (binary_switch/455994)
[5/22/2017, 9:27:14 AM] [Wink] Update sent successfully: Kitchen Nook Ceiling (light_bulb/2645178)
[5/22/2017, 9:27:18 AM] [Wink] Update sent successfully: Kitchen Main (light_bulb/2645199)
[5/22/2017, 9:55:37 AM] [Wink] Refreshing devices...
[5/22/2017, 9:55:39 AM] [Wink] Devices refreshed
```

**Comment by sibartlett** [comment]
That's an issue I've been dealing with myself. PubNub push notifications stop working for a period of time. I'm working on code to detect this situation, and then re-initiatize push notifications.

**Comment by sibartlett** [comment]
I'm closing this issue, as I've opened a new issue which explains the problem in more detail: #15

**Comment by sibartlett** [comment]
I've published a new version `1.4.0` which hopefully fixes the issue - though I'm still testing it myself.

**Comment by sibartlett** [comment]
The fix in `1.4.0` didn't work. I've now released another version (`1.4.2`) and will monitor it to see if it works this time.
